### PR TITLE
Export without argument

### PR DESCRIPTION
**Pull request by t-h2o**

### CODE REVIEW

**Review by lgyger — COMMENTED** [review]
You should check if the environment is empty then, simply return so that there is no uneccessary malloc. Otherwise well done for creating such a clear, efficient and well structured code

File: srcs/export_no_arg.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+// Print export
+void	print_export(char **envp)
+{
+	char	**table;
+	int		tmp;
+	int		n;
+
+	n = count(envp);
```
Comment (lgyger) [inline srcs/export_no_arg.c:86]:
```suggestion
	int		n;
        if (!envp || !envp[0])
            return ;
	n = count(envp);
```

File: srcs/export_no_arg.c [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		++i;
+	if (s1[i] == 0)
+		return (1);
+	if (s1[i] < s2[i])
+		return (0);
+	return (1);
+}
+
+// Swap the address
+static void	swap(char **a, char **b)
```
Comment (lgyger) [inline srcs/export_no_arg.c:46]:
```suggestion
static inline void	swap(char **a, char **b)
```

**Review by lgyger — CHANGES_REQUESTED** [review]
checked the environment and added an inline

File: srcs/export_no_arg.c (hunk 1 above)
Comment (t-h2o) [inline srcs/export_no_arg.c:86]:
```suggestion
    if (!envp || !envp[0])
            return ;
	n = count(envp);
```

File: srcs/export_no_arg.c (hunk 2 above)
Comment (t-h2o) [inline srcs/export_no_arg.c:46]:
```suggestion
static inline void	swap(char **a, char **b)
```

**Review by t-h2o — COMMENTED** [review]
Redo the same review

**Pull request merged**